src/main/java/org/openstreetmap/atlas/geography/atlas/AtlasResourceLoader.java

```diff
@@ -54,6 +54,18 @@
         }
     }
 
+    /**
+     * @author lcram
+     */
+    private class AlwaysTrueAtlasFilter implements Predicate<Resource>
```

Comment:
Nit: Instead of a class, why not just a `private final Predicate<Resource> alwaysTrueAtlasFilter = resource -> return true;`

Reply:
Ah ok this is the syntax. I couldn't figure out how to make the compiler stop complaining and got frustrated so I just made a class :P

Reply:
Also slight addendum
`private final Predicate<Resource> alwaysTrueAtlasFilter = resource -> return true;` actually doesn't seem to be working. It looks like it needs to be
`private final Predicate<Resource> alwaysTrueAtlasFilter = resource -> {return true; };`

Reply:
No IDE to help me on github :) - glad you were able to make it work!